README.md

```diff
@@ -12,7 +12,7 @@ Conductor Mobile is a port of the [Conductor](https://github.com/conductor-frame
     <dependency>
         <groupId>com.willowtreeapps</groupId>
         <artifactId>conductor-mobile</artifactId>
-        <version>0.19.2</version>
+        <version>0.19.3</version>
```

Comment:
Should this be 0.20?

Reply:
You da best, fixed in fa56888b59e7a52855ad425b8d4ae8ee1bbe1040